src/Yaapii.Atoms/Collection/Sorted.cs

```diff
@@ -38,7 +38,7 @@ public sealed class Sorted<T> : CollectionEnvelope<T>
         /// A list with default sorting (ascending)
         /// </summary>
         /// <param name="src">the source enumerable</param>
-        public Sorted(params T[] src) : this(new ListOf<T>(src))
+        public Sorted(params T[] src) : this(new List.List.Live<T>(src))
```

Comment:
We must not use a namespace which is the same as a class name - this leads to conflicts in our projects.
The current solution is, that we use the plural for the namespace. Texts, Lists, Collections - or another name if applicable (see map area: "Lookup"

Reply:
Renaming the namespace results in a breaking change. Should ListOf and Sticky be removed then completly too? Users have to change their code anyway due to the namespace renaming.

Reply:
When we have the current change ready, the switch from live to sticky by default will be a big BC also, so we will release Atoms as version 2.0 when we are done. Until then, no new Atoms package will be released.

Reply:
I removed Live and Sticky, we should provide an upgrade documentation instead.